feeds/wifi-trunk/hostapd/files/hostapd.sh

```diff
@@ -338,7 +336,6 @@ hostapd_set_bss_options() {
 	set_default chan_util_avg_period 600
 	set_default utf8_ssid 1
 	set_default multi_ap 0
-	set_default rssi_reject_assoc_rssi 0
```

Comment:
Why is rssi_reject_assoc_rssi getting removed?

Reply:
Because of the following commit: a9b3afd. It did not properly fix the issues with **hostapd** on **ecw5211** device.